src/Container/ApplicationFactory.php

```diff
@@ -17,6 +17,7 @@
 use Zend\Expressive\Exception;
 use Zend\Expressive\Router\Aura as AuraRouter;
 use Zend\Expressive\Router\RouterInterface;
+use Zend\Expressive\Router\Route;
```

Comment:
sort the use statement, place `Zend\Expressive\Router\Route` before `Zend\Expressive\Router\RouterInterface`